server/middleware/auth.js

```diff
@@ -91,4 +91,27 @@ export default class auth {
     }
     return next();
   }
+
+  /**
+   * authorizes a user whose role is set to "superAdmin"
+   *
+   * @static
+   * @param {*} req
```

Comment:
It's advised you describe you add a description of a function parameter in jsdoc for better readability of your code.